docs/batch.rst

```diff
@@ -124,7 +124,9 @@ Users may need to customize the script to meet their requirements.
 
     "OCR_INPUT_DIRECTORY", "Set input directory to monitor (recursive)"
     "OCR_OUTPUT_DIRECTORY", "Set output directory (should not be under input)"
+    "OCR_ARCHIVE_DIRECTORY", "Set archive directory for processed originals (should not be under input, requires ``OCR_ON_SUCCESS_ARCHIVE`` to be set)"
     "OCR_ON_SUCCESS_DELETE", "This will delete the input file if the exit code is 0 (OK)"
+    "OCR_ON_SUCCESS_ARCHIVE", "This will move the processed orignal file to ``OCR_ARCHIVE_DIRECTORY`` if the exit code is 0 (OK). Note that ``OCR_ON_SUCCESS_DELETE`` takes precedence over this option, i.e. if both options are set, the input file will be deleted."
```

Comment:
I'd prefer to see this transformed into `OCR_ON_SUCCESS` with options being `"keep"`, `"delete"` or `"archive"`. If `"OCR_ON_SUCCESS_DELETE=1"`, then we set `OCR_ON_SUCCESS=delete` and print a deprecation warning that `OCR_ON_SUCCESS_DELETE` is deprecated. With your proposed change, the choice is no longer boolean.

Reply:
That's what I figured as well, I just didn't want to do deprecate existing stuff without discussing it first. I'll happily change it to a single option with three possible values.